package.json

```diff
@@ -51,7 +51,8 @@
 		"type-check": "tsc --noEmit"
 	},
 	"dependencies": {
-		"@typescript-eslint/utils": "^5.62.0"
+		"@typescript-eslint/scope-manager": "^6.21.0",
+		"@typescript-eslint/utils": "^6.21.0"
```

Comment:
Shouldn't this be:
```
"@typescript-eslint/utils": "^8.12.0"
```

?

Reply:
No, because this pull request is upgrading them to v6, not v8

Reply:
Ah! I see there is a https://github.com/testing-library/eslint-plugin-testing-library/pull/947 v7 and presumably v8 will happen at some point.